pkg/plugins/platform/templates.go

```diff
@@ -69,10 +69,8 @@ services:
     image: {{ .router_image }}:{{ .router_tag }}
     container_name: nginx-proxy
     ports:
-      - "80:80"
-      - {{ .mailhogport }}:{{ .mailhogport }}
-      - {{ .dbaport }}:{{ .dbaport }}
-      - {{ .dbport }}:{{ .dbport }}
+      {{ range $port := .ports }}- "{{ $port }}:{{ $port }}"
```

Comment:
Are you sure about that? It seems to me that we would want a 1:1 relation here. If a user defined `VIRTUAL_PORT=8765`, the router would create an upstream listening on port 8765 internally, and we would then want to the router to accept external traffic from 8765.